html/pfappserver/root/static.alt/src/views/Nodes/_components/NodeView.vue

```diff
@@ -142,14 +230,20 @@
 
         <b-tab title="Option82">
           <template slot="title">
-            {{ $t('Option82') }}
+            {{ $t('Option82') }} <b-badge pill v-if="node.dhcpoption82 && node.dhcpoption82.length > 0" variant="light" class="ml-1">{{ node.dhcpoption82.length }}</b-badge>
           </template>
+          <b-table stacked="sm" :items="node.dhcpoption82" :fields="dhcpOption82Fields" v-if="node.dhcpoption82" striped></b-table>
```

Comment:
We should probably create a new component for *dhcpoption82* since we have a searchable API endpoint.

Reply:
Agreed, I'll defer on writing the component for now and will revisit this again later.